certification/internal/policy/operator/deployable_by_olm.go

```diff
@@ -65,6 +65,10 @@ func (p *DeployableByOlmCheck) initClient() error {
 	if err := openshift.AddSchemes(scheme); err != nil {
 		return fmt.Errorf("could not add new schemes to client: %w", err)
 	}
+
+	// TODO(): GetConfig generates a rest config from environment paths. We already have
```

Comment:
This might be what you are looking for: https://github.com/test-network-function/cnf-certification-test/pull/644